### DISCUSSION TITLE
How to release new version for this gem?

### DISCUSSION THREAD

**Discussion by PikachuEXE** [redis-store/redis-store#366] [General]
Although https://github.com/redis-store/redis-store/pull/359 related but I think it's better to start a new conversation to not spam other people

As I said in https://github.com/redis-store/redis-store/pull/359#issuecomment-1710977008

> The [publish workflow](https://github.com/redis-store/redis-store/blob/master/.github/workflows/publish.yml) seems to be using `RUBYGEMS_API_KEY` which does not require me being gem owner

So I am not sure what should be done to release this gem (just add a commit with version updated & tag it with `v` prefix?)
Or should I even be able to publish new versions (I have no idea what are the expectations for new collaborators here

@radar

**Comment by radar** [comment]
I think we need to bump the version (see #367) and then we can run the publish workflow.
